askama_derive/src/config.rs

```diff
@@ -574,7 +574,7 @@ mod tests {
                     "::askama::Html".into()
                 ),
                 (
-                    str_set(&["md", "none", "txt", "yml", ""]),
+                    str_set(&["md", "none", "txt", "yml", "typ", ""]),
```

Comment:
Seems okay. Mind keeping them in alphabetic order?

Reply:
It seems fine to me, I change it.